external/common_functions.sh

```diff
@@ -14,7 +14,7 @@
 #
 
 # Supported versions
-supported_versions="8 9 10 11 12 13 14 15 16 17"
+supported_versions="8 9 10 11 12 13 14 15"
```

Comment:
You can change this line back to include the 16 and 17 (my branch was older and we want to keep this change).

Reply:
Done